android/room/app/build.gradle

```diff
@@ -0,0 +1,56 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "csc780.sfsu.edu.roompersistencelibrary"
```

Comment:
Please change package name to org.androidtutorials.roompersistencelibrary. Also refactor all package name accordingly.

Reply:
Actually, change the name to org.androidtutorials.room, since that is what you called the directory.